brave/brave-core
# Fixes users are not served ads due to failure to initialize Brave Ads when migrating to database schema 21 or above

Resolves https://github.com/brave/brave-browser/issues/22540

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm ads are served after upgrading from 1.36.x, see https://github.com/brave/brave-browser/issues/22540 for pre-requisites before upgrading
- Confirm existing `creative_ad_conversion` entries are migrated when upgrading from a build before the fix
